### PR TITLE
Add the Made by Observable button to the top nav

### PR DESCRIPTION
**Pull request by CobusT**
This PR adds the Made by Observable component to the Plot site.

### CODE REVIEW

File: docs/.vitepress/theme/CustomLayout.vue [hunk 1]
```diff
@@ -35,4 +42,20 @@ const {Layout} = DefaultTheme;
   background-color: rgba(37, 37, 41, 0.5);
 }
 
+/* Hide the dark mode toggle */
+.VPNavBarAppearance,
+.VPNavBarExtra,
+.VPNavBarHamburger {
+  display: none !important;
+}
```
Comment (mbostock) [inline docs/.vitepress/theme/CustomLayout.vue:50]: This shouldn’t be necessary with `appearance: "force-auto"`?

Comment (mbostock) [reply docs/.vitepress/theme/CustomLayout.vue:50]: Or maybe the comment only applied to `.VPNavBarAppearance`, whereas we still also want to hide `.VPNavBarExtra` and `.VPNavBarHamburger`? Although it makes me wonder what functionality we are losing by hiding those latter two things?

Comment (CobusT) [reply docs/.vitepress/theme/CustomLayout.vue:50]: Yup, `.VPNavBarAppearance` is not needed anymore (removed it). In terms of what we are missing from the othhers, they are actually empty with our implementation.

**Review by mbostock — APPROVED** [review]
Looks great!

**Pull request merged**